scripts/publish-crates.sh

```diff
@@ -37,6 +36,7 @@ published_crates=(
   "biome_css_factory"
   "biome_css_parser"
   "biome_deserialize"
+  "biome_formatter"
```

Comment:
biome_formatter needs to be released after publishing biome_deserialize

Reply:
Oooh , so the order matters in this file! Good to know